phpstan/config.example_install_for_phpstan.php

```diff
@@ -0,0 +1,149 @@
+<?php
```

Comment:
Why is there the need for another config file that is 99.99% the same as the other config file?

Reply:
Phpstan config file contains only configuration, install/example config file contains some additional check for maintenance etc. Also in install/example config there is null for smtp settings, which results in false positives (constant is always null instead of dynamic string).

Reply:
I see, then it is ok I guess.